.gitpod-files/coolwsd-gitpod.xml

```diff
@@ -3,6 +3,10 @@
     <!-- Note: 'default' attributes are used to document a setting's default value as well as to use as fallback. -->
     <!-- Note: When adding a new entry, a default must be set in WSD in case the entry is missing upon deployment. -->
 
+    <accessibility desc="Accessibility settings">
+        <enable type="bool" desc="Controls whether accessibility support should be enabled or not." default="false">true</enable>
```

Comment:
We do not use this file in CI or in production. You can leave the setting `false`.

Reply:
Okay, anyway I think it doesn't make any difference since it's pretty ignored